package/ports_daemon/ports_daemon/src/serial.c

```diff
@@ -85,13 +85,13 @@ static int uart_configure(const uart_t *uart)
 {
   int fd = open(uart->tty_path, O_RDONLY | O_NONBLOCK);
   if (fd < 0) {
-    piksi_log(LOG_ERR, "error opening tty device");
+    piksi_log(LOG_ERR, "error opening tty device for path %s", uart->tty_path);
```

Comment:
We should add LOG_SBP to the piksi_log call so that these will get surfaced in HITL:
```
piksi_log(LOG_ERR|LOG_SBP, "error opening tty device for path %s", uart->tty_path);
```